napari/_vispy/vispy_image_layer.py

```diff
@@ -16,11 +17,33 @@
 ]
 
 
-class VispyImageLayer(VispyBaseLayer):
-    def __init__(self, layer):
+class ImageLayerNode:
```

Comment:
I'm fine with this too, very straight forward

Reply:
Yes it feels like some bigger change is afoot there. But this was all I needed. To start I just put `TiledImageVisual` under the empty `VispyBaseLayer.node` but that seemed likely to cause some problem. Having `self.node` but the `TiledImageVisual` itself seemed clearly better/cleaner.